modules/network/lang.json

```diff
@@ -0,0 +1,56 @@
+{
+  "en": {
+    "create_subnet": "Create Subnet",
+    "cidr_tip": "Please fill in the blank matching format such as 192.168.1.0/24",
+    "enable_security": "Enable security restrict(Once be set, it can not be changed)",
+    "enable_vlan": "Vlan network",
+    "external_network": "External Network",
```

Comment:
the key external_network is duplicate in 2 files:
halo/client/applications/admin/modules/network-agents/lang.json
halo/client/applications/admin/modules/network/lang.json